Azure/communication-ui-library-android
# [Feature] Localization custom string apply initial flow

## Purpose

* Custom String API implement for LocalizationConfiguration
* Custom String apply in SetupControlBarView with localizationProvider dependency injection in order to test the views in future

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
CallCompositeJavaLauncher.java class has a custom map passed while creating 
LocalizationConfiguration instance. Please check for the mic on/off button to see 
effect of the custom string. 
```

## What to Check
Verify that the following are valid
* If a string map of specific <key, value> is provided with localizationConfiguration the library would have override the texts no matter what language is chosen from the settings menu

## Other Information
